### PR TITLE
SumList Fix + UT

### PR DESCRIPTION
**Pull request by teizenman**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20221

### CODE REVIEW

File: Scripts/SumList/SumList.py [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
-elif isinstance(value, str):
-    try:
-        result = int(value)
-        demisto.results(result)
-    except TypeError:
-        return_error('The string does not represent a number.')
-
-else:
-    return_error('This transformer applies only to a list of numbers.')
+            value = [int(item) for item in value.split(',')]
```
Comment (ronykoz) [inline Scripts/SumList/SumList.py]: what about floats?

Comment (teizenman) [reply Scripts/SumList/SumList.py]: Fixed

File: Scripts/SumList/SumList.py [hunk 2]
```diff
@@ -1,33 +1,37 @@
...
-        else:
-            try:
-                result = int(value[0])
-                demisto.results(result)
-            except TypeError:
-                return_error('The string does not represent a number.')
-    else:
+
+def sum_value(value):
+    if isinstance(value, int):
```
Comment (ronykoz) [inline Scripts/SumList/SumList.py]: Again, it could be a float

File: Scripts/SumList/SumList.py [hunk 3]
```diff
@@ -1,33 +1,37 @@
...
+            return 'error', 'error'
+
+    elif isinstance(value, list):
+        try:
+            value = [float(item) for item in value]
+        except ValueError:
+            return 'error', 'error'
+
+    result = sum(value)
+    if result == int(result):
```
Comment (ronykoz) [inline Scripts/SumList/SumList.py]: Why do we need this?

Comment (teizenman) [reply Scripts/SumList/SumList.py]: Just to prettify the result, but it doesn't really matter

**Pull request merged**